angularjs/angular.d.ts

```diff
@@ -1698,8 +1698,9 @@ declare module angular {
         /**
          * Controller constructor function that should be associated with newly created scope or the name of a registered
          * controller if passed as a string. Empty function by default.
+         * Use the array form to define dependencies (necessary if strictDi is enabled and you require dependency injection)
          */
-        controller?: string | Function;
+        controller?: string | Function | any[];
```

Comment:
We can add more improvement.  `string | Function | (string | Function)[];`
http://goo.gl/9yIghI